src/Pages/MyPage/MyPageComponent/ClassCard.js

```diff
@@ -0,0 +1,173 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { FaRegHeart, FaHeart, FaThumbsUp, FaFireAlt } from "react-icons/fa";
+import styled from "styled-components";
+const ClassCard = ({
+  cardWidth,
+  imgSrc,
+  isOpening,
+  category,
+  mentor,
+  description,
+  likeCount,
+  cheeredRate,
+  thumbsUp,
+  originPrice,
+  discount,
+  month,
+}) => {
+  const [isLiked, setIsLiked] = useState(false);
+  const [pricePerMonth, setPricePerMonth] = useState(0);
+  useEffect(() => {
+    if (month && originPrice && discount) {
+      const discountedPrice = originPrice * discount;
+      setPricePerMonth(discountedPrice / month);
+    }
+  }, [discount, originPrice, month]);
+  return (
```

Comment:
```suggestion
  const [pricePerMonth, setPricePerMonth] = useState(0);
  
  useEffect(() => {
    if (month && originPrice && discount) {
      const discountedPrice = originPrice * discount;
      setPricePerMonth(discountedPrice / month);
    }
  }, [discount, originPrice, month]);
  
  return (
```
별도의 로직들 끼리는 한줄씩 띄워서 구분해주세요

Reply:
ClassCard.js는 팀원이 작업했던 컴포넌트가 아직 merge가 안됐어서 잘 붙여지는지 보려고 일단 따로 파일 받아서 임포트했습니다! 현재는 merge되어서 저 파일은 지우고 머지된 파일로 임포트 경로 수정했습니다!